boards/arm/nucleo_f429zi/nucleo_f429zi.dts

```diff
@@ -47,6 +47,9 @@
 		led1 = &blue_led_1;
 		led2 = &red_led_1;
 		sw0 = &user_button;
+		/* Temp provide arduino i2c alias */
+		/* Workaround waiting meta-zephyr-sdk/issues/92 resolution */
```

Comment:
Do we still need this, since the SDK now supports a good version of DTC?

Reply:
Good catch! Not needed anymore indeed.
Fixed, thanks!